FlashLFQ/FlashLfqEngine.cs

```diff
@@ -201,7 +201,7 @@ public FlashLfqResults Run()
             _results.CalculatePeptideResults();
 
             // do top3 protein quantification
```

Comment:
Comment need to be updated.

Reply:
thx, good catch. it's not necessary to keep both. I think median polish would be better than top3 in the vast majority of scenarios. There's no particular reason to keep both methods around and I will probably eventually delete the top3 code but I'd rather be careful right now